### DISCUSSION TITLE
Full stream title when mouseover a followed channel

### DISCUSSION THREAD

**Discussion by Morsheeva** [night/betterttv#6452] [Q&A] [answered]
I wish we'd be able to see the complete title of a stream when we mouseover a followed channel. i didn't find any option about this, do someone know how i could do this ? Thank you.

**Comment by night** [comment, marked as answer]
Twitch shows the full stream title in the native tooltip when you hover over a followed channel in the sidebar — no BTTV-specific setting is needed for this. Closing as answered.
